### PR TITLE
Better portal configurability and docs

### PR DESCRIPTION
**Pull request by franciscop-sc**
- Added classname to the profile popup to allow styling
- Made the dropdown profile position depend on the size of the profile
- Added docs
- Lock should never be published in a library
- Added docs for `createGlobalStyle` since classes somehow had lower preference when imported as `./style.css`

### CODE REVIEW

File: src/Profile.js [hunk 1]
```diff
@@ -49,12 +54,18 @@ const Box = styled.div`
     height: 0;
     border: 8px solid black;
     top: -8px;
-    right: 20px;
+    /* width / 3 = width of img */
+    /* width / 3 - 16px (self-width) - 2px (margin) */
```
Comment (mayt) [inline src/Profile.js:58]: del?

File: src/Profile.js [hunk 2]
```diff
@@ -49,12 +54,18 @@ const Box = styled.div`
...
+    right: calc(33.33% / 2 - 14px);
     border-width: 0 8px 8px 8px;
     border-color: transparent transparent #ccc transparent;
   }
 `;
 
+const P = styled.p`
```
Comment (mayt) [inline src/Profile.js:65]: not sure why but html tag names for styled components field weird to me. But it should not be a problem

**Review by mayt — APPROVED** [review]

File: src/Profile.js (hunk 1 above)
Comment (franciscop-sc) [reply src/Profile.js:58]: Oh these are comments on purpose to explain where this came from

File: src/Profile.js (hunk 2 above)
Comment (franciscop-sc) [reply src/Profile.js:65]: Ah I see, no problem calling it "Name" or sth.

Comment (franciscop-sc) [reply src/Profile.js:65]: Fixed 👍

**Pull request merged**